stdlib/Random/src/Random.jl

```diff
@@ -27,7 +27,7 @@ export rand!, randn!,
        shuffle, shuffle!,
        randperm, randperm!,
        randcycle, randcycle!,
-       AbstractRNG, MersenneTwister, RandomDevice
+       AbstractRNG, MersenneTwister, RandomDevice, TaskLocal, Xoshiro
```

Comment:
Should `TaskLocal` really be exported? If so, I think it should be called `TaskLocalRNG`, as the name `TaskLocal` sounds very non-specific and doesn't give a hint that it's related RNGs.